src/pages/2022-gartner-cool-vendors-in-software-engineering.tsx

```diff
@@ -20,7 +20,7 @@ export const Report: FunctionComponent = () => {
                     variant="venusCode"
                     title="Cool Vendors in Software Engineering: Enhancing Developer Productivity"
                     leftCol={
-                        <div className="col-12 px-0 pb-2 pb-lg-0">
+                        <div className="col-12 tw-px-0 tw-pb-xxs lg:tw-pb-0">
```

Comment:
```suggestion
                        <div className="col-lg-5 col-4 tw-px-0 tw-pb-xxs lg:tw-pb-0">
```
Weirdly this logo gets really big on this page now. The img height below isn't being applied. 🤔 These grid columns help it, unless you have a different idea
![image](https://user-images.githubusercontent.com/59381432/184262399-53bc5c3b-ada8-4918-8b4e-fd421d05e40f.png)

Reply:
Ok so I ended up swapping this out for using grid instead, and I also restricted the props so it's not open to ambiguous flexibility. So now the hero component follows only the variants listed in the DLS Hero component. Spacing is already pre-defined in the Hero and TwoColumnSection, so whenever you want to pass a div/element or set of elements to `leftColumn`, we don't have to add any margins. The only thing that would be custom would be vertical alignment on mobile/desktop if you need that for a template. I added a `mergeColumns` prop to conjoin/justify the left/right columns together for scenarios like the webinar hero and the Gartner report hero. You can check out [4a9c9c7](https://github.com/sourcegraph/about/pull/5654/commits/4a9c9c794a06426709ec50a54b1d7ce87f56e8f9) but I smoked tested the pages using our Hero and also pages using the TwoColumnSection to make sure `reverseOnMobile` works as intended.